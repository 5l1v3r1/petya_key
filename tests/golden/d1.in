rzvj43fWvf5ef3Nq6W7Z4Zbzoehmh557m972QhMedVHQvLyju3mx7BrRb5XpkvD21JUVw1M9Cs74AvZpzGTWp4xbp9aVqsoL
