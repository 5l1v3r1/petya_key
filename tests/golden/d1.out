f63117737990bd58de1d91a63a9156a0
